src/domain/entities/User.ts

```diff
@@ -1,18 +1,40 @@
 import _ from "lodash";
+import { FutureData } from "./Future";
 import { NamedRef } from "./Ref";
 
-export interface User {
+export interface OrgUnitAccess {
+    orgUnitId: string;
+    orgUnitName: string;
+    readAccess: boolean;
+    captureAccess: boolean;
+}
+
+export interface UserGroup {
+    id: string;
+    name: string;
+}
+export interface ModuleAccess {
+    moduleId: string;
+    moduleName: string;
+    readAccess: boolean;
+    captureAccess: boolean;
+    usergroups: UserGroup[];
+}
+
+export interface UserAccessInfo {
     id: string;
     name: string;
     username: string;
     userRoles: UserRole[];
     userGroups: NamedRef[];
+    userOrgUnitsAccess: OrgUnitAccess[];
+    userModulesAccess: FutureData<ModuleAccess[]>;
```

Comment:
That's not right, a entity structure should not hold a Future. It should be `ModuleAccess[]`.

Reply:
Addressed in https://github.com/EyeSeeTea/glass-dev/pull/35/commits/9e3f96184e88bbae9ca572610f2973d8fc45a599